### PR TITLE
Bowtie Info on Homepage

### PR DESCRIPTION
**Pull request by Siddharth-Singh-2004**
fix-#368 to show a description of what Bowtie is, on the homepage

----
📚 Documentation preview 📚: https://bowtie-json-schema--885.org.readthedocs.build/en/885/

### CODE REVIEW

File: frontend/src/components/BowtieInfo/BowtieInfoSection.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        <table className="table table-sm table-hover">
+          <thead>
+            <tr>
+              <td>
+                Bowtie is tool for understanding and comparing implementations
+                of the &nbsp;
+                <a
+                  href={"https://json-schema.org/"}
+                  target={"_blank"}
+                  rel={"noopener noreferrer"}
```
Comment (Julian) [inline frontend/src/components/BowtieInfo/BowtieInfoSection.tsx:15]: It's fine to include `referrer` here, linking to the JSON Schema site is essentially "internal" inasmuch as we probably would be interested to know if lots of traffic was being generated from Bowtie's site.

```suggestion
                  rel={"noopener"}
```

File: frontend/src/components/BowtieInfo/BowtieInfoSection.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                  {"JSON Schema specification"}
+                </a>{" "}
+                &nbsp; across all programming languages. The report below uses
+                the &nbsp;
+                <a
+                  href={
+                    "https://github.com/json-schema-org/JSON-Schema-Test-Suite"
+                  }
+                  target={"_blank"}
+                  rel={"noopener noreferrer"}
```
Comment (Julian) [inline frontend/src/components/BowtieInfo/BowtieInfoSection.tsx:26]: And here, same.
```suggestion
                  rel={"noopener"}
```

File: frontend/src/components/BowtieInfo/BowtieInfoSection.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                >
+                  {"official JSON Schema Test Suite"}
+                </a>{" "}
+                &nbsp; to display bugs or functionality gaps in implementations.
+                You can also use Bowtie to get this information about your own
+                schemas. You can find out how in its &nbsp;
+                <a
+                  href={"https://docs.bowtie.report/"}
+                  target={"_blank"}
+                  rel={"noopener noreferrer"}
```
Comment (Julian) [inline frontend/src/components/BowtieInfo/BowtieInfoSection.tsx:36]:
```suggestion
                  rel={"noopener"}
```

And certainly in Bowtie's own docs.

**Comment by Julian** [comment]
Thanks! Just some notes about not scrubbing referrers, but this looks good afterwards, appreciated!

**Comment by Siddharth-Singh-2004** [comment]
@Julian,

I presume we want links to json schema.org, test suite etc., **to open in a new tab/window** based on user's browser preferences. If that is not a requirement, then I just remove the "target=_blank" and you don't have to read any further. Else, there are some nuances and I need your concurrence to make the changes.

If I use "a href" tag with target={"_blank"}, then nox's  "ui(style)" checker throws up errors when rel="noopener" is used without rel="noreferrer" (don't understand why it is so).

Here's a screenshot of the error when code is such..
 <a
                  href={"https://docs.bowtie.report/"}
                  target={"_blank"}
                  rel={"noopener"}
 >

Errors:
Screenshot 2024-02-17 at 9 38 20 PM

Ask from you:
I have two choices, either fix nox checks for "ui(style)" to not mandate "rel=noreferrer" (when target is used, not investigated if this is supported in nox, but will in due course) or remove target option from <a href>. 

Please let me know how you want me to proceed.

**Comment by Julian** [comment]
We may not care about such old browsers but I'm less inclined to do the research right this second -- it's fine to just remove the `target: _blank` entirely and have it load in the same tab then!

**Comment by Siddharth-Singh-2004** [comment]
Thanks for the quick revert. I'll have the fix in shortly.

**Comment by Julian** [comment]
Great! Think this looks good! Merging, thanks again!

**Pull request merged**